### PR TITLE
Anya/getting sql

### PR DESCRIPTION
**Pull request by atbida**

### CODE REVIEW

File: docs/low-code-sql/getting-started-with-low-code-sql.md [hunk 1]
```diff
@@ -0,0 +1,382 @@
+---
```
Comment (several27) [inline docs/low-code-sql/getting-started-with-low-code-sql.md:1]: @atbida Could we move this under the `Getting Started` section?

Comment (several27) [reply docs/low-code-sql/getting-started-with-low-code-sql.md:1]: We can have two sections there - `Spark with Databricks` & `SQL with Databricks`

File: docs/low-code-sql/getting-started-with-low-code-sql.md [hunk 2]
```diff
@@ -0,0 +1,382 @@
...
+  - tutorial
+---
+
+In this quick-start, we will show you how to setup Prophecy low-code SQL with an existing Databricks warehouse.
+
+Requirements:
+
+- Databricks Account
+- GitHub Account (optional)
+
```
Comment (several27) [inline docs/low-code-sql/getting-started-with-low-code-sql.md:17]: Would you mind writing here a little bit of an introduction?

Comment (several27) [reply docs/low-code-sql/getting-started-with-low-code-sql.md:17]: Also, are we going to be able to share the video here?

File: docs/low-code-sql/getting-started-with-low-code-sql.md [hunk 3]
```diff
@@ -0,0 +1,382 @@
...
+Here we create customers_nations model that’s going to enrich our customers and produce a report of which customers show up in which geographic areas most commonly.
+
+To create a new model simply click on the **(1) + Add Model** in the sidebar, when hovering over Models section, or **(2) Create Model** button. A model creation pop-up will show up, with very similar options, as when we defined the seed, available. Enter the **(3) Model Name** and **(4) Model Path** and that’s it. Finally save the model by pressing **(5) OK.**
+
+### 7.1 Drag and drop model’s graph
+
+<div class="wistia_responsive_padding" style={{padding:'56.25% 0 0 0', position:'relative'}}>
+<div class="wistia_responsive_wrapper" style={{height:'100%',left:0,position:'absolute',top:0,width:'100%'}}>
+<iframe src="https://Prophecy-1.wistia.com/medias/wowmg8l0qq" title="Drag and Drop Models Graph" allow="autoplay; fullscreen" allowtransparency="true" frameborder="0" scrolling="no" class="wistia_embed" name="wistia_embed" msallowfullscreen width="100%" height="100%"></iframe>
+</div></div>
```
Comment (several27) [inline docs/low-code-sql/getting-started-with-low-code-sql.md:245]: Screenshot 2023-06-05 at 9 41 59 PM

There seems to be something off with the rendering here. Wistia logo shows up at the top

File: docs/low-code-sql/img/7-2-build-model.mp4
Comment (several27) [inline docs/low-code-sql/img/7-2-build-model.mp4:1]: What is this video used for @atbida? Let's remove if not, we should avoid large files on git :P

**Review by several27 — CHANGES_REQUESTED** [review]

**Review by several27 — APPROVED** [review]

**Review by keerthiprophecy — APPROVED** [review]

**Pull request merged**